src/tools/genpolicy/src/policy.rs

```diff
@@ -347,6 +360,9 @@ pub struct RequestDefaults {
     /// Commands allowed to be executed by the Host in all Guest containers.
     pub ExecProcessRequest: ExecProcessRequestDefaults,
 
+    /// Interfaces allowed to be updated to in all Guest sandboxes.
```

Comment:
Nit: the settings are mostly about what _not_ to allow updating, so the comment is a bit off.